src/Components/Body/styles.ts

```diff
@@ -0,0 +1,69 @@
+import {StyleSheet} from 'aphrodite';
+
+const styles = StyleSheet.create({
+    
+    body: {
+        height: '80vh',
+        padding: 7,
+        boxSizing: 'border-box',
+        zIndex: 1,
+    },
+    h1: {
+        textAlign: 'center',
+    },
+    main_instructions: {
+        textAlign: 'center',
+        fontSize: '18px',
+    },
+
+    form: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+    },
+
+    label: {
+        display: 'flex',
+        fontSize: 'large',
+        fontWeight: 'bold',
+        margin: '5px',
```

Comment:
use number instead of string
like this margin: 5

Reply:
done